lib/spack/spack/test/installer.py

```diff
@@ -442,6 +471,8 @@ def test_package_id_ok(install_mockery):
     assert pkg.name in inst.package_id(pkg)
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
I would expect tests involving locking, binary caches, and upstreams to fail, but this one seems like it should pass since it's fairly basic.

Reply:
It passes. Amended.